tests/unit/test_utils_filesystem.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shutil
+
+import pytest
+
+from pip._internal.utils.filesystem import copy2_fixed, is_socket
+
+from ..lib.filesystem import make_socket_file, make_unreadable_file
+from ..lib.path import Path
```

Comment:
```suggestion
from tests.lib.path import Path
```

nit: The absolute import is clearer IMO.

Reply:
[Updated](https://github.com/pypa/pip/pull/6844/files#diff-e5e3005ef6c051e6287f93d9c1c721e3R9).